velox/exec/tests/AggregationTest.cpp

```diff
@@ -1230,7 +1232,9 @@ TEST_F(AggregationTest, groupingSets) {
 
   // Compute a subset of aggregates per grouping set by using masks based on
   // group_id column.
-  plan = PlanBuilder()
+  // comments this case, because we change group_id column in
```

Comment:
@zhejiangxiaomai Can you help to remove the change about gid calculations? It seems no need this change. Thanks.

Reply:
https://github.com/oap-project/gluten/pull/1309 will verify “ remove the change about gid calculations”